scripts/openstack-fake-update.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+set -ex
+OUTFILE=${OUTFILE:-csv.yaml}
+TIMEOUT=${TIMEOUT:-300s}
+VERSION=${VERSION:-0.0.2}
+
+if [ -z "$OPERATOR_NAMESPACE" ]; then
+    echo "Please set OPERATOR_NAMESPACE"; exit 1
+fi
+if [ -z "$NAMESPACE" ]; then
+    echo "Please set NAMESPACE"; exit 1
+fi
+
+OPENSTACK_OPERATOR_CSV=$(oc get csv -n $OPERATOR_NAMESPACE -o name | grep openstack-operator)
+oc get $OPENSTACK_OPERATOR_CSV -o yaml -n $OPERATOR_NAMESPACE  > $OUTFILE
+sed -i $OUTFILE -e "s|value: .*/podified-antelope-centos9/\(.*\)@.*|value: quay.io/podified-antelope-centos9/\1:current-podified|g"
```

Comment:
We should be able to have this as variable.  So in the makefile we should consume some variables "from" and "to" that will be passed down to the script.  Those variables can then be defined in the job and passed down by ci-framework.  A first approximation could be to have that as the default.

Reply:
"from" variable comes from deployment and it is not needed to run this script. We consume whatever we get from already deployed csv. Why would we need "from" variable ?
"to" variable added.